src/core_plugins/kibana/public/dashboard/listing/dashboard_listing.html

```diff
@@ -100,7 +100,7 @@
     <!-- PromptForItems -->
     <div
       class="kuiPanel kuiPanel--centered kuiPanel--withHeader"
-      ng-if="!listingController.isFetchingItems && !listingController.items.length && !listingController.filter"
+      ng-if="!listingController.isFetchingItems && !listingController.items.length && !listingController.filter && !listingController.hideWriteControls"
```

Comment:
This should probably still be displayed even when `listingController.hideWriteControls === true`, but sans the `Let's create some!` bit.